hack/common.sh

```diff
@@ -658,6 +658,25 @@ EOF
 }
 readonly -f os::build::save_version_vars
 
+# os::build::get_product_vars exports variables that we expect to change
+# depending on the distribution of Origin
+function os::build::get_product_vars() {
+  if [[ "${OS_ROOT}" == *origin* ]]; then
```

Comment:
Hrm, I was expecting the spec file to specify the variable that triggers this if block.  Then someone can test without moving their git repo directory.